.planning/codebase/STRUCTURE.md

```
@@ -1,127 +1,199 @@
 # Codebase Structure
 
-**Analysis Date:** 2026-08-03
+**Analysis Date:** 2026-08-30
 
 ## Directory Layout
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)